Tomas Heran <tomas.heran@oracle.com>
Typo in configure.ac when pkg-config is used to find GnuTLS library.
Upstream bug: https://gnunet.org/bugs/view.php?id=4888

diff -r 1cf9eb13b637 -r 8bd4ec7772bc configure.ac
--- a/configure.ac	Tue Feb 14 13:06:13 2017 +0100
+++ b/configure.ac	Tue Feb 14 13:06:35 2017 +0100
@@ -1025,7 +1025,7 @@
           [
            AC_MSG_RESULT([[yes]])
            have_gnutls=yes
-           GNUTLS_CPPLAGS="$GNUTLS_CFLAGS"
+           GNUTLS_CPPFLAGS="$GNUTLS_CFLAGS"
            GNUTLS_LDFLAGS="$GNUTLS_LIBS"
            AC_MSG_CHECKING([[for gnutls_privkey_import_x509_raw()]])
            AC_LINK_IFELSE([
